openshift-install-powervs

```diff
@@ -1186,7 +1186,21 @@ function setup_poweriaas() {
 # If latest is available in System PATH then use symlink
 #-------------------------------------------------------------------------
 function setup_ibmcloudcli() {
-  CLI_LATEST=$(curl -s https://api.github.com/repos/IBM-Cloud/ibm-cloud-cli-release/releases/latest | grep tag_name | cut -d'"' -f4 | sed 's/v//')
+  TAGS=( $(git -c 'versionsort.suffix=-' ls-remote --exit-code --refs --sort='version:refname' --tags \
+  https://github.com/IBM-Cloud/ibm-cloud-cli-release '*.*.*') )
+
+  OS=$(uname -s); 
```

Comment:
Why uname? Did you check what it returns on other operating systems eg: windows?

Reply:
Also OS is global variable you don't want to override it here.